blocks/fragment/fragment.js

```diff
@@ -22,7 +22,8 @@ export async function loadFragment(path) {
     const resp = await fetch(`${path}.plain.html`);
     if (resp.ok) {
       const main = document.createElement('main');
-      main.innerHTML = await resp.text();
+
+      main.innerHTML = DOMPurify.sanitize(await resp.text());
```

Comment:
Note that the original `fragment` block source is in the [block-collection](https://github.com/adobe/aem-block-collection/tree/main/blocks/fragment) and should probably be fixed there first, then updated here.